mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -12,11 +12,24 @@
 
 from mitosheet.transpiler.transpile_utils import param_dict_to_code
 
+from mitosheet.excel_utils import get_column_from_column_index
+
 # This is a helper function that generates the code for formatting the excel sheet
 def get_format_code(state: State) -> list:
     code = []
     formats = state.df_formats
-    for sheet_name, format in zip(state.df_names, formats):
+    for sheet_index, (sheet_name, format) in enumerate(zip(state.df_names, formats)):
+        # We need to convert the column IDs to column letters for conditional formats to export to excel
+        conditional_formats = format.get('conditional_formats')
+        column_ids = state.column_ids.get_column_ids(sheet_index=sheet_index)
+        if conditional_formats is not None:
+            for conditional_format in conditional_formats:
+                # Create new object to store the columns in the excel format
+                conditional_format['columns'] = []
```

Comment:
Rather than editing the conditional formatting object, let's make a new object. In it, we can include:
1. The columns (you do this)
2. The filter condition and values (so just copying this over)

This should be pretty simple, but will protect us from accidentally editing objects we're using elsewhere in our codebase with fields we don't expect. 

I think you're on the right track with what to pass to the `add_formatting_to_excel_sheet` function, though! We def want to give it columns rather than column ids or indexes or anything -- users will find that so clear.

Reply:
Sounds good!

Reply:
Added that in, how does it look?